content/courses/mobile/intro-to-solana-mobile.md

```diff
@@ -1215,15 +1223,15 @@ on the `main` branch of the repository.
 
 ## Challenge
 
-Your challenge today is to take our app and add a decrement function. Simply add
-another button and call the `decrement` function on our program. This
-instruction already exists on the program and its IDL, so you simply need to
-write client code to call it.
-
-After you give it a try on your own, feel free to take a look at the
-[solution code on the `solution` branch](https://github.com/Unboxed-Software/solana-react-native-counter/tree/solution).
-
-<Callout type="success" title="Completed the lab?">
-Push your code to GitHub and
-[tell us what you thought of this lesson](https://form.typeform.com/to/IPH0UGz7#answers-lesson=c15928ce-8302-4437-9b1b-9aa1d65af864)!
-</Callout>
+Your next challenge is to expand the app by adding a `decrement` function. You
+need to create another button that will call the `decrement` method on the
+Solana program. The logic for the decrement function already exists in the
+program’s **IDL** (**Interface Description Language**), so your task is to write
+the client-side code that interacts with it.
+
+Once you've completed this, you can check your solution against the solution
+code available on the
+[solution branch](https://github.com/Unboxed-Software/solana-react-native-counter/tree/solution).
+If you’ve successfully completed the lab, push your code to GitHub and share
+your feedback on this lesson through this
```

Comment:
Keep the callout please.

Reply:
i have included it but i noticed that `<Callout>` doesn't allow the links to render properly (safe to say it won't enable markdown syntax, it might be an issue with my text editor)